lxd-agent/main_agent.go

```diff
@@ -320,10 +324,10 @@ func (c *cmdAgent) mountHostShares() {
 
 		_, err = shared.RunCommand("mount", args...)
 		if err != nil {
-			logger.Errorf("Failed mount %q (Type: %q, Options: %v) to %q: %v", mount.Source, mount.FSType, mount.Options, mount.Target, err)
+			l.Error("Failed to mount", logger.Ctx{"err": err})
```

Comment:
For this specific error, I think that having the mount args (or some of them) could be handy.

Reply:
Fixed. Also ive just noticed the hot plug path doesn't seem to have support for options like the start time ones do :(